app/src/test/kotlin/fr/free/nrw/commons/utils/ImageUtilsTest.kt

```diff
@@ -0,0 +1,15 @@
+package fr.free.nrw.commons.utils
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class AlgoCheck {
+    @Test
+    fun validateImageForOkImage() {
+        assertEquals(ImageUtils.IMAGE_OK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/check1.jpg"))
+        assertEquals(ImageUtils.IMAGE_OK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/check2.jpg"))
+        assertEquals(ImageUtils.IMAGE_OK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/check3.jpg"))
+        assertEquals(ImageUtils.IMAGE_OK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/check4.jpg"))
+        assertEquals(ImageUtils.IMAGE_DARK, ImageUtils.checkIfImageIsTooDark("src/test/resources/fr/free/nrw/commons/utils/check5.jpg"))
```

Comment:
This line should probably be moved to its own test, called something like `validateImageForDarkImage`